src/github.com/getlantern/errlog/errlog.go

```diff
@@ -63,6 +63,29 @@ type systemInfo struct {
 	OSArch    string `json:"osArch"`
 }
 
+var (
+	defaultSystemInfo *systemInfo
+	userLocale        *UserLocale
+)
+
+func init() {
+	version, _ := osversion.GetHumanReadable()
+
+	defaultSystemInfo = &systemInfo{
+		OSType:    runtime.GOOS,
+		OSArch:    runtime.GOARCH,
+		OSVersion: version,
+	}
+
+	lang, _ := jibber_jabber.DetectLanguage()
+	country, _ := jibber_jabber.DetectTerritory()
+	userLocale = &UserLocale{
```

Comment:
`userLocale` can change during Lantern lifecycle. When application code does log error with locale (which should happen rarely because most code doesn't care about locale), then the information should be accurate.